texts/epn_paper.md

```diff
@@ -10,6 +10,8 @@
 - Babak Seradjeh (Indiana University)
 - Daniel Varjas (Delft University of Technology)
 
+> Virtual Science Forum is an open, collaborative and volunteer-run effort to accommodate organization of online conferences. Originally initiated by physicist from different backgrounds and locations aiming to provide a service to the research community.
```

Comment:
```suggestion
> Virtual Science Forum is an open, collaborative, and volunteer-run effort to facilitate online scientific seminars.  Originally initiated by physicists from different backgrounds and locations, we aim to provide a service and make a positive impact on the research community.
```

Reply:
Another remark here: we're providing a bio, so let's start with "The authors are some of the members of..."

Also let's expand "different" to be more specific—we likely have space.

Reply:
```suggestion
> The authors are some of the contributors of Virtual Science Forum, an open, collaborative and volunteer-run effort to facilitate online scientific seminars. Originally initiated by physicists with backgrounds ranging from condensed matter to quantum computation, located all over Europa and United states with the aim to provide a service and and make a positive impact on the research community.
```

Reply:
Updated my suggested change to reflect your remarks.

Reply:
@vfatemi somehow the final
> we aim to make...
doesn't read natural with the 3rd person used at the beginning of the bio.

Reply:
Right. We should pick either 1st or 3rd person overall. I checked recent EPN articles and they are all in third person, so let's do that.

Reply:
>  The authors are members of the Virtual Science Forum, an open, collaborative, and volunteer-run effort to facilitate online scientific seminars. They work at academic institutions spread across North America and Europe and donate their effort to make a positive impact on the research community.